generic-database-plugin/docs/Database-action.md

```diff
@@ -14,42 +14,27 @@ For example, you may want to run a sql update command on a database before the p
 
 Properties
 ----------
-**Database Command:** The database command to execute.
-
-**Host:** host where database running on.
+**Driver Name:** Name of the JDBC driver to use.
 
-**Port:** database port.
+**Connection String:** JDBC connection string including database name.
 
-**Database:** database name.
+**Database Command:** Database command to execute.
 
 **Username:** User identity for connecting to the specified database.
 
 **Password:** Password to use to connect to the specified database.
 
-**Driver Name:** Name of the JDBC driver to use.
 
 **Connection Arguments:** A list of arbitrary string tag/value pairs as connection arguments. These arguments
 will be passed to the JDBC driver, as connection arguments, for JDBC drivers that may need additional configurations.
-This is a semicolon-separated list of key-value pairs, where each pair is separated by a equals '=' and specifies
-the key and value for the argument. For example, 'key1=value1;key2=value' specifies that the connection will be
-given arguments 'key1' mapped to 'value1' and the argument 'key2' mapped to 'value2'. (Macro-enabled)
 
 Example
 -------
-
-    {
-        "name": "Database",
-        "plugin": {
-            "name": "Database",
-            "type": "action",
-            "properties": {
-                "query": "UPDATE table_name SET price = 20 WHERE ID = 6",
-                "host": "localhost",
-                "port": 3306,
-                "database": "prod",
-                "user": "user123",
-                "password": "password-abc",
-                "jdbcDriverName": "mysql"
-            }
-        }
-    }
+Execute database statement "UPDATE table_name SET price = 20 WHERE ID = 6" against database "prod" running on localhost,
```

Comment:
Use complete sentences in the documentation.

Suppose you want to execute a query against a PostgreSQL database named "prod" that is running on "examplehost" port 5432. Ensure that the driver for PostgreSQL is installed, then configure the plugin with:

```
Driver Name: "postgres"
...
```

Reply:
updated